face_track/face_atomic.py

```diff
@@ -32,7 +32,9 @@ def __init__(self):
 		self.port = 17020
 
 	def update_face_octomap(self, faceid, xx, yy, zz):
-		face="(map-ato \"faces\" (NumberNode \""+str(faceid)+"\") "+str(xx)+" "+str(yy)+" "+str(zz)+")"
+		face="(map-ato \"faces\" (NumberNode \""+str(faceid)+"\") "+str(xx)+" "+str(yy)+" "+str(zz)+")\n"
+		netcat(self.hostname, self.port, face + "\n")
+		face = "(NumberNode \"" + str(faceid) + "\" (av 5 0 0))\n"
```

Comment:
you could just put the av up above.  Also, number nodes don't need quotes. So this would work:
```
face="(map-ato \"faces\" (NumberNode "+str(faceid)+" (av 5 0 0)) "+str(xx)+" "+str(yy)+" "+str(zz)+")"
```